viewers/pydicom_PIL.py

```diff
@@ -25,36 +23,44 @@
 # Tested on Python 2.5.4 (32-bit) on Mac OS X 10.6
 #    using numpy 1.3.0 and PIL 1.1.7b1
 
+
 try:
-    import PIL.Image
+    from PIL import Image,ImageEnhance
     have_PIL = True
 except ImportError:
     have_PIL = False
 
+#apply_modality_lut() function comes with pydicom module
 try:
-    import numpy as np
-    have_numpy = True
+    from pydicom.pixel_data_handlers.util import apply_modality_lut
+    have_lut = True
 except ImportError:
-    have_numpy = False
+    have_lut = False
 
 
-def get_LUT_value(data, window, level):
+def get_LUT_value(dataset):
     """Apply the RGB Look-Up Table for the given
-       data and window/level value."""
-    if not have_numpy:
-        raise ImportError("Numpy is not available."
-                          "See http://numpy.scipy.org/"
+       dataset"""
+    if not have_lut:
+        raise ImportError("pydicom is not available."
+                          "See https://pydicom.github.io/pydicom/dev/tutorials/installation.html"
                           "to download and install")
 
-    return np.piecewise(data,
-                        [data <= (level - 0.5 - (window - 1) / 2),
-                         data > (level - 0.5 + (window - 1) / 2)],
-                        [0, 255, lambda data: ((data - (level - 0.5)) /
-                         (window - 1) + 0.5) * (255 - 0)])
+    #Pixel Data
+    pixel_data = dataset.pixel_array
+
+    return apply_modality_lut(pixel_data,dataset)
+
 
+def get_PIL_image(dataset,brightness_factor = 1.0):
+    """Get Image object from Python Imaging Library(PIL)
```

Comment:
Just style, but should be space after comma and no spaces around the = for a function argument if not using typing notation